From bf6a58cdc1ae177986d42be72cb2c3b5b4abf466 Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Sat, 28 Dec 2019 01:43:18 +0100
Subject: [PATCH 70/98] dt-bindings: vendor-prefixes: Add prefix for
 "mxtronics"

Beijing Times Minxin Technology Co., Ltd. [0] offers various integrated
circuits such as (but not limited to):
- Beidou / GPS / Glonass multi-module basebands
- Audio D/A converters
- USB to serial converters
- 4bit and 8bit MCUs

[0] http://www.mxtronics.com/

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 Documentation/devicetree/bindings/vendor-prefixes.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/vendor-prefixes.yaml b/Documentation/devicetree/bindings/vendor-prefixes.yaml
index c7591b2aec2a74..57fb871f76f5ca 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.yaml
+++ b/Documentation/devicetree/bindings/vendor-prefixes.yaml
@@ -1102,6 +1102,8 @@ patternProperties:
     deprecated: true
   "^mxicy,.*":
     description: Macronix International Co., Ltd.
+  "^mxtronics,.*":
+    description: Beijing Times Minxin Technology Co., Ltd.
   "^myir,.*":
     description: MYIR Tech Limited
   "^national,.*":

